gatsby-node.esm.js

```diff
@@ -308,9 +317,18 @@ exports.createPages = async ({ actions, graphql }) => {
             },
           })
         })
+      } else {
+        // No need to create referrer URLs 
+        createPage({
+          path: `${path}/${data.causeId}/`,
+          component: causeLaunchData.preview ? HomePageWrapper : NotFoundPage,
+          context: {
+            data: pivotedData,
+          },
+        })
       }
-    )
-  }
+    }
+  )
```

Comment:
I think this is missing this acceptance criterion: "If vertical has a preview page enabled and the vertical has already launched, it does not render content and redirects to the real landing page for the vertical. Client-side redirect is OK." Currently, the preview page would 404, right? The goal with the redirect is to provide content to any people who are still using the preview link for whatever reason.

BTW, the [Gatsby S3 plugin](https://www.gatsbyjs.com/plugins/gatsby-plugin-s3/) that we use might help with redirects. But if it's not immediately easy to work in this case, just go with a client-side redirect.

Reply:
Ohh, yes I just missed that.